extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -627,9 +690,7 @@ def validate(self, classifier_data):
             '_w_dp',
             '_l_dp',
             '_c_dl',
-            '_c_lw'
-        ]
-        list_of_list_bool_properties = [
+            '_c_lw',
             '_b_dl'
```

Comment:
A lot seems to have changed since I last reviewed this PR. I think I mentioned the fact that LDAStringClassifier might (or rather *will*) become useless as things stand. In the light of this fact, I don't really understand why are we still fixing the LDAStringClassifier. I think it is really good to bring up the issues that exist so that we know what can be improved, but, can we please limit this PR just to fixing the classification pipeline? (i.e. the original intent of the issue: using cached classifier instead of generating new one every time)